subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmTestSuite.java

```diff
@@ -40,7 +40,11 @@
     void useJUnitJupiter(String version);
     void useJUnit();
     void useJUnit(String version);
-    // TODO: Spock, TestNG?
+    void useSpock();
+    void useSpock(String version);
+    void useKotlinTest();
+    void useKotlinTest(String version);
+    // TODO: TestNG?
```

Comment:
Thank you for adding! You will likely have to rebase this after #18343 is merged; sorry!